content/guides/figma/getting-started.mdx

```diff
@@ -1,10 +1,11 @@
 ---
 title: Getting started
+description: Principles, standards, and usage guidelines for designing GitHub interfaces with Figma
 ---
 import {LinkExternalIcon} from '@primer/octicons-react'
 
 ## What are Primer Figma libraries?
-The Primer Figma libraries contain UI components and design tokens (styles) that our teams at GitHub use to design GitHub. The components contained within Primer match what is available for developers in Primer React Components, Primer ViewComponents, and Primer CSS.
+The Primer Figma libraries contain UI components and design tokens (`variables` and `styles`) that our teams at GitHub use to design GitHub. The components contained within Primer match what is available for developers in Primer React Components, Primer ViewComponents, and Primer CSS.
```

Comment:
If we refer to libraries as plural I would like to seen a list of which ones exist and what's inside them.

Reply:
Fix and I only refer to Primer web now. Great catch.